server/routes/api.js

```diff
@@ -14,7 +14,7 @@ router.get('/articles', function (req, res, next) {
             res.json(articles)
         }
     };
-    articleClient.findArticles({}, callBack)
+    articleClient.findArticles({ 'visible': true }, callBack)
```

Comment:
no need for `'` around `visible` since it's one word (and it doesn't have dashes or spaces)